File: src/main/java/ast/ASTElseStmnt.java
package ast;

public class ASTElseStmnt extends ASTNode {
    public ASTNodeArray<ASTNode> code;

    public ASTElseStmnt(ASTFileLocation loc, ASTNodeArray<ASTNode> code){
        super(loc);
        this.code = code;
    }

    public void print(int i){
        printIndent(i);
        System.out.println("ASTElseStmnt: ");
        code.print(i+1);
    }
}

File: src/main/kotlin/ast/ASTElseStmnt.kt
package ast

class ASTElseStmnt(loc: ASTFileLocation, val code: ASTNodeArray<ASTNode>) : ASTNode(loc), ASTBlockStmnt {

    override fun print(i: Int) {
        printIndent(i)
        println("ASTElseStmnt: ")
        code.print(i + 1)
    }

    override fun getBlock(): ASTNodeArray<ASTNode> {
        return code
    }
}